Lib/subprocess.py

```diff
@@ -642,7 +644,6 @@ def __init__(self, args, bufsize=-1, executable=None,
         self.stderr = None
         self.pid = None
         self.returncode = None
-        self.universal_newlines = universal_newlines
```

Comment:
The `universal_newlines` instance attribute is part of the public API of the class (even though not documented).

We should keep the name around.  Even if it is turned into a property with a setter and getter that ultimately sets/returns the value of the new `self.text_mode` attribute.